lib/utilities/run-command.js

```diff
@@ -18,6 +23,11 @@ module.exports = function run(/* command, args, options */) {
   }
 
   debug("running command=" + command + "; args=" + args + "; cwd=" + process.cwd());
+
+  if (isWindows && command === 'npm') {
+    return exec(command + ' ' + args.join(' '));
```

Comment:
True. I could remove this fix from the PR and appveyor would start failing. I figured I would write the failing test and the fix in one go.